documentation/src/main/asciidoc/introduction/Introduction.adoc

```diff
@@ -111,7 +111,7 @@ Most people implement the domain model as a set of what we used to call "Plain O
 
 [TIP]
 ====
-Take your time with this code, and try to produce a Java model that's as close as reasonable to the relational data model. Avoid using exotic or advanced mapping features when they're not really needed.
+Take your time with this part, and try to produce a Java model that's as close as reasonable to the relational data model. Avoid using exotic or advanced mapping features when they're not really needed.
```

Comment:
not sure why you propose this change.

Reply:
When I saw "this code", my first reaction is some concrete code snippet, so it confused me. I have no objection to revert it back for sure.